types/types.go

```diff
@@ -103,7 +103,8 @@ const (
 	// DefaultStaleReplicaTimeout in minutes. 48h by default
 	DefaultStaleReplicaTimeout = "2880"
 
-	EngineImageChecksumNameLength = 8
+	EngineImageChecksumNameLength          = 8
+	InstanceManagerImageChecksumNameLength = 8
```

Comment:
We can use a single const `ImageChecksumNameLength` instead.

Reply:
Fixed